staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -95,6 +95,7 @@ func createHandler(r rest.NamedCreater, scope *RequestScope, admit admission.Int
 		}
 
 		options := &metav1.CreateOptions{}
+		options.TypeMeta.SetGroupVersionKind(metav1.SchemeGroupVersion.WithKind("CreateOptions"))
```

Comment:
I expected these to get set in the options structs *after* we decoded parameters and validated them (e.g. line ~110).

Reply:
Same comment applies to all operations (especially delete, in lieu of needing to set on a deep copy)

Reply:
Oops, yes, fixing

Reply:
Oops, yes, fixing